admiral/envs/components/actor.py

```diff
@@ -230,6 +230,50 @@ def process_move(self, agent, acceleration, angle, **kwargs):
         self.position.modify_position(agent, np.array([x_position, y_position]))
         return agent.position - position_before
 
+class AccelerationMovementActor:
+    """
+    Process x,y accelerations for VelocityAgents, which are given an 'accelerate'
+    action. Update the agents' positions based on their new velocity.
+
+    position_state (ContinuousPositionState):
+        The position state handler. Needed to modify agent positions.
+    
+    velocity_state (VelocityState):
+        The velocity state handler. Needed to modify agent velocities.
+    
+    agents (dict):
+        The dictionary of agents.
+    """
+    def __init__(self, position_state=None, velocity_state=None, agents=None, **kwargs):
```

Comment:
Thinking about space applications (also drones or submarines) we could add a dimension variable (standard = 2) and then use it in the action space. Wouldn't hurt here, I guess, and open up for potential cool applications.

Reply:
Captured in #46